examples/gno.land/r/demo/foo20/foo20_test.gno

```diff
@@ -54,3 +54,43 @@ func TestReadOnlyPublicMethods(t *testing.T) {
 		}
 	}
 }
+
+func TestErrConditions(t *testing.T) {
+	admin := std.Address("g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj")
+	empty := std.Address("")
+
+	type test struct {
+		name string
+		msg  string
+		fn   func()
+	}
+
+	std.TestSetOrigCaller(admin)
+	{
+		tests := []test{
+			{"Transfer(admin, 1)", "cannot send transfer to self", func() { Transfer(a2u(admin), 1) }},
+			{"Approve(empty, 1))", "invalid address", func() { Approve(a2u(empty), 1) }},
+		}
+		for _, tc := range tests {
+			shouldPanicWithMsg(t, tc.fn, tc.msg)
+		}
+	}
+}
+
+func a2u(addr std.Address) users.AddressOrName {
+	return users.AddressOrName(addr)
+}
```

Comment:
I think this function is not needed. I prefer using `users.AddressOrName()` directly. 

If you prefer to keep it, maybe use a better name. And replace all occurrences of `users.AddressOrName()`

Reply:
I'll follow origin style which removes `a2u()`